cumulusci/tasks/preflight/sobjects.py

```diff
@@ -17,6 +18,48 @@ def _run_task(self):
         )
 
 
+class CheckSObjectPerms(BaseSalesforceApiTask):
+    task_options = {
+        "permissions": {
+            "description": "The object permissions to check. Each key should be an sObject API name, under which Boolean describe values can be specified, "
+            "for example, Account: queryable: True. The output is True if all sObjects and permissions are present and matching the specification.",
```

Comment:
Could the example include two permissions to check under the `Account` sObject? I'm guessing permissions are separated by commas if you want to list more than one for an sObject? Maybe could include that in the description as well.

Reply:
I added fuller task docs for the tasks in this module to address this.